### ISSUE TITLE
ZipException: archive is not a ZIP archive java.lang.reflect.InvocationTargetException

### ISSUE THREAD

**Issue by Abhisek1994** [spring-projects/spring-ide#225]
Unable to launch File>New>Spring Starter Project

Sort of an emergency. Please help.

It is showing-

'Import Getting Started Content' has encountered a problem.
ZipException: archive is not a ZIP archive

Details->
ZipException: archive is not a ZIP archive
java.lang.reflect.InvocationTargetException

**Comment by martinlippert** [comment]
What happens if you go to http://start.spring.io, create your project there, download the ZIP file, extract it, and import via "Import as Existing Maven Project..." ? Does that work?

**Comment by s24963386** [comment]
I had the same problem. when i go to http://start.spring.io,create and downd ,then import,it wok ok.

**Comment by s24963386** [comment]
![image](https://user-images.githubusercontent.com/1725974/34336523-ce2dbd40-e992-11e7-9498-2a324c8776e9.png)

**Comment by martinlippert** [comment]
@s24963386 can you take a look at the Error Log view to see if there are more details, like a full stack trace that is related to this?

**Comment by kdvolder** [comment]
Do you have some kind of proxy / firewall in your network environment? It could be a bug related to proxy handling, or... it could be proxy needs to be configured properly in Eclipse preferences (it doesn't allways follow OS-wide settings).

**Comment by kdvolder** [comment]
Interesting, both http and https urls are serving the same content, so it is bit strange. It probably means there's some difference in how some kind of proxy / firewall treats http versus https traffic in your environment.

**Comment by chairmo** [comment]
I'm encountering the same error on STS 4 - 4.5.0 when i used Gradle 3x as build tool yet Gradle 3x integration is installed by default. 
I went to http://start.spring.io, created a new project, downloaded the ZIP file, extracted it, and imported via "Import as Existing Gradle Project..." It still doesn't run.

Please, note: When I use Maven as a build tool it runs fine.

**Comment by martinlippert** [comment]
@chairmo This sounds different from the description above (where downloading the project and importing it into the IDE works). Can you share the exact error message?

**Comment by chairmo** [comment]
![sts4](https://user-images.githubusercontent.com/18530636/71935055-272fd600-31a6-11ea-871b-8a3fa8763368.JPG)

the error is encountered only when Gradle build tool is used on starting >spring starter project and when importing the extracted zip folder. STS 4 - 4.5.0

**Comment by martinlippert** [comment]
Is there anything related showing up in the `Error Log` view that might reveal more details, like a full stack trace?

**Comment by Nirmala9955** [comment]
> What happens if you go to http://start.spring.io, create your project there, download the ZIP file, extract it, and import via "Import as Existing Maven Project..." ? Does that work?

yes sir this is working but how to solve the problem in eclipse only, because it is a time taking approach, if you have any solution, please suggest me, I will be more helpful, THANK you sir

**Comment by kdvolder** [comment]
> yes sir this is working but how to solve the problem in eclipse only, because it is a time taking approach, if you have any solution, please suggest me, I will be more helpful, THANK you sir

We can try to help. But we have some questions:

-  what is the error that you get exactly? Also look in the Eclipse error log which you can open vie "Window >> Show View >> Other >> Error Log". 
- does your network or computer have proxies and/or firewalls that might be blocking network traffic from STS?

**Comment by Nirmala9955** [comment]
I got this log messages

**Comment by martinlippert** [comment]
@Nirmala9955, I don't see a log message attached here. Would be good to get some insights into the Error Log view messages to see more details here. Please try again to attach the content of the error message.
